src/common/storage/backends/StorageClient.js

```diff
@@ -87,5 +70,11 @@ define([
         throw new Error(`stat not implemented for ${this.name}`);
     };
 
+    StorageClient.prototype.checkStreamsInBrowser = async function() {
```

Comment:
Personally, I would rename this to something like `ensureStreamsSupported` or `ensureIsBrowser(msg)`. This doesn't actually check streams and it appears that streaming in the browser is partially supported (`getStream` appears to be supported).

Reply:
Also, this shouldn't be `async`